bowtie/_cli.py

```diff
@@ -1357,6 +1359,32 @@ async def filter_dialects(
             break
 
 
+@subcommand
+@click.option(
+    "--dialect",
+    "-D",
+    "dialect",
+    type=_Dialect(),
+    default=lambda: max(Dialect.known()),
+    show_default=True,
+    metavar="URI_OR_NAME",
+    help=(
+        "A URI or a shortname identifying the dialect of a test report. "
+        f"Possible shortnames include: "
+        f"{', '.join(sorted(Dialect.by_alias()))}"
+    ),
+)
+def latest_report(dialect: Dialect):
+    """
+    Output latest Bowtie report for a given dialect.
+
+    If no dialect provided, it outputs Bowtie's latest
+    dialect's latest report.
+    """
+    for line in asyncio.run(dialect.latest_report()).iter_lines():
```

Comment:
You don't really need lines here, you're just writing the contents unchanged -- possibly there's something even more straightforward that isn't occurring to me but at least the below makes fewer assumptions:

```python
def latest_report(...):
    async def write(response):
        async for chunk in (await response).aiter_bytes():
            click.echo(chunk)
    asyncio.run(write(dialect.latest_report()))

```